apps/passport-client/src/worker/service-worker.ts

```diff
@@ -53,6 +59,18 @@ const EPHEMERAL_CACHE_RESOURCES = new Set([
   ...GENERATED_CHUNKS
 ]);
 
+/**
+ * Z cache is for Z API-relevant resources which do not change frequently, such
```

Comment:
I'm not sure it's accurate to extend this to other "ZAPI-relevant resources" given the reliance on GPC artifacts NPM version below.  That makes it pretty specific to GPC artifacts, though I guess if the same approach were taken for a different type of artifact, the cache would simply clear when either of the versions changed.